collectors/build/jenkins-audit/src/test/java/com/capitalone/dashboard/collector/AuditCollectorTaskTest.java

```diff
@@ -0,0 +1,86 @@
+package com.capitalone.dashboard.collector;
+
+import com.capitalone.dashboard.collector.config.FongoConfig;
+import com.capitalone.dashboard.collector.config.TestConfig;
+import com.capitalone.dashboard.model.AuditException;
+import com.capitalone.dashboard.model.AuditResult;
+import com.capitalone.dashboard.model.AuditType;
+import com.capitalone.dashboard.model.Dashboard;
+import com.capitalone.dashboard.repository.AuditResultRepository;
+import com.capitalone.dashboard.repository.DashboardRepository;
+import com.capitalone.dashboard.repository.ComponentRepository;
+import com.capitalone.dashboard.repository.CollectorRepository;
+import com.capitalone.dashboard.repository.CollectorItemRepository;
+import com.capitalone.dashboard.response.DashboardReviewResponse;
+import com.capitalone.dashboard.service.DashboardAuditService;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+
+import java.io.IOException;
+import java.util.*;
+
+import static junit.framework.TestCase.assertNotNull;
+import static org.mockito.Mockito.mock;
+
+
+@RunWith(SpringJUnit4ClassRunner.class)
+@ContextConfiguration(classes = {TestConfig.class, FongoConfig.class})
+public class AuditCollectorTaskTest {
+    private AuditResultRepository mockstatusRepository;
+    @Autowired
+    private DashboardAuditService dashboardAuditService;
+    @Autowired
+    private DashboardRepository dashboardRepository;
+    @Autowired
+    private ComponentRepository componentRepository;
+    @Autowired
+    private CollectorRepository collectorRepository;
+    @Autowired
+    private CollectorItemRepository collectorItemRepository;
+
+    @Before
+    public void setup() throws IOException {
+        mockstatusRepository = mock(AuditResultRepository.class);
+        TestUtils.loadDashBoard(dashboardRepository);
+        TestUtils.loadCollector(collectorRepository);
+        TestUtils.loadComponent(componentRepository);
+        TestUtils.loadCollectorItems(collectorItemRepository);
+        AuditCollectorSettings settings = new AuditCollectorSettings();
+        settings.setServers(Arrays.asList("http://localhost:8081/"));
```

Comment:
servers should come from properties

Reply:
This is unit test class, we were ok with it.